src/apis/utils.ts

```diff
@@ -1,16 +1,15 @@
-export const fetchJson = (url, option) => fetch(url, {
-    ...option,
-    headers: {
-      ...option.headers,
-      'Content-Type': 'application/json;',
-    },
-  })
-  .then(response => {
-    if (response.status < 400) {
-      return response.json()
-    }
-    throw response
-  })
+export const fetchJson = (url, option) => {
+    const {headers, ...reset} = option
+    return fetch(url, {
+        ...reset,
```

Comment:
passed wrong params to fetch. And Only login and logout need  to contain 'Content-Type': 'charset=utf-8', others are only 'application/json; '